test/source/tests/compose.ts

```diff
@@ -1497,6 +1497,13 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
       await expectRecipientElements(composePage, { to: ['flowcrypt.compatibility@gmail.com', 'vladimir@flowcrypt.com'], cc: [], bcc: [] });
     }));
 
+    ava.default('compose - reply - from === acctEmail, not the first message in a thread, its subject starts with Re:', testWithBrowser('compatibility', async (t, browser) => {
+      const appendUrl = 'threadId=17d02296bccd4c5d&skipClickPrompt=___cu_false___&ignoreDraft=___cu_false___&replyMsgId=17d02296bccd4c5d';
```

Comment:
I see threadId = '17d02296bccd4c5d', and messageId = '17d02296bccd4c5d'. Does this mean that this message is actually the first in the thread?

Reply:
Correct, it's the first one in the mock, but with `Re: ` in its subject, so it acts like the response in the thread.

Reply:
Ok, can we name the test as `compose - reply - subject starts with Re:` as this is the only thing that is tested here from my understanding?

Reply:
Done 👍